rfc/system/RFC-4570-flyte-auth-in-task.md

```diff
@@ -0,0 +1,145 @@
+# [RFC Template] Flyte Authentication in Tasks
+
+**Authors:**
+
+- @zeryx
+
+## 1 Executive Summary
+
+This RFC proposes a method to flow Flyte authentication information into task containers. 
+It aims to simplify access to execution-specific information and enable secure, streamlined usage of Flyte features within tasks. 
+The proposal introduces a Flyte Auth Agent system to bind the authentication context of the task/workflow executor to all executed tasks.
```

Comment:
What would be the `task/workflow executor` here? I'm thinking the user who triggered the workflow?